doc/developer_guide.md

```diff
@@ -22,6 +22,30 @@ Then make sure your PATH includes the `~/.local/bin` directory, e.g. with:
 
     export PATH="$PATH":~/.local/bin
 
+### Pre-commit hooks
+
+GitHub pre-commit hooks are scripts that can be triggered automatically by
```

Comment:
I suppose you wanted to talk about git hooks in general, not only "pre-commit" hook, here. If so, please remove the word "pre-commit".

Also, it should be "Git", not "GitHub".